src/main/resources/application.yml

```diff
@@ -107,3 +107,31 @@ spring:
 
 server:
   port: 8080
+
+---
+spring:
+  datasource:
+    driver-class-name: org.h2.Driver
+    username:
+    password:
+    url: jdbc:h2:./src/main/resources/data/appuser;AUTO_SERVER=TRUE
```

Comment:
please use the proper h2 in-memory settings

Reply:
in-memory or embedded because there are properties for embedded.